frontend/packages/integration-tests-cypress/tests/app/debug-pod.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { checkErrors, testName } from '../../support';
+import { detailsPage } from '../../views/details-page';
+import { listPage } from '../../views/list-page';
+import { modal } from '../../views/modal';
+import { nav } from '../../views/nav';
+import * as yamlEditor from '../../views/yaml-editor';
+
+const POD_NAME = `pod1`;
+const CONTAINER_NAME = `container1`;
+const podToDebug = `apiVersion: v1
+kind: Pod
+metadata:
+  name: pod1
+spec:
+  containers:
+    - name: container1
+      image: fedora
```

Comment:
It's better if we can use an image from quay to avoid Docker Hub rate limiting in CI.